spinflip/spin_flip_tools.py

```diff
@@ -161,36 +178,57 @@ def H_R_pm(self, theta, phi):
             for z in range(self.nz)]).transpose((1,2,0))
         return  np.average(H_R_pm, axis = 2)
     
-    #derivative term (doesn't account for zsim gradient)
+    #derivative terms (don't account for zsim gradient, since its approximately negligible)
```

Comment:
Should these two functions be combined into one? (e.g. H_grad with a L or R argument)

Reply:
I could! just to avoid code duplicatioN?